kitsune/users/static/users/js/stores/UserAuthStore.es6.js

```diff
@@ -1,126 +0,0 @@
-/* globals _:false */
```

Comment:
Referenced in kitsune/questions/static/questions/js/actions/AAQActions.es6.js
which is referenced in... lots of places... I think all the react has to go.